src/tools/skylark/javatests/com/google/devtools/skylark/skylint/DocstringUtilsTest.java

```diff
@@ -28,6 +28,45 @@
 /** Tests the {@link DocstringUtils} class. */
 @RunWith(JUnit4.class)
 public class DocstringUtilsTest {
+  @Test
+  public void emptyDocstring() throws Exception {
+    List<DocstringParseError> errors = new ArrayList<>();
+    DocstringInfo info = DocstringUtils.parseDocstring("", 0, errors);
+    Truth.assertThat(info.summary).isEmpty();
+    Truth.assertThat(info.parameters).isEmpty();
+    Truth.assertThat(info.longDescription).isEmpty();
+    Truth.assertThat(errors.toString())
+      .contains("1: the docstring is empty");
+
+    errors = new ArrayList<>();
+    info = DocstringUtils.parseDocstring("\n", 0, errors);
+    Truth.assertThat(info.summary).isEmpty();
+    Truth.assertThat(info.parameters).isEmpty();
+    Truth.assertThat(info.longDescription).isEmpty();
+    Truth.assertThat(errors.toString())
+      .contains("1: the docstring is empty");
+  }
+
+  @Test
+  public void missingOneLineSummary() throws Exception {
+    List<DocstringParseError> errors = new ArrayList<>();
+    DocstringInfo info = DocstringUtils.parseDocstring("\n\n", 0, errors);
+    Truth.assertThat(info.summary).isEmpty();
+    Truth.assertThat(info.parameters).isEmpty();
+    Truth.assertThat(info.longDescription).isEmpty();
+    Truth.assertThat(errors.toString())
+      .contains("1: the one-line summary is missing");
+
+
+    errors = new ArrayList<>();
+    info = DocstringUtils.parseDocstring("\nfoo\n", 0, errors);
```

Comment:
So this now raises an Issue?

    def foo():
        """
        Does a foo.
        """

This seems like a separate stylistic discussion that should be in another PR. Is there any corresponding github issue / consensus?

Reply:
i uses the format of skydoc (https://skydoc.bazel.build/docs/writing.html). the doc format should be matching.

Reply:
@brandjon